app/assets/stylesheets/application.scss

```diff
@@ -126,4 +126,45 @@ h6 {
 
 .nav-feature {
   color: black;
+
+}
+
+.center-img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.item-show-img {
+  width: auto;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+
+}
+
+.item-background-img {
+
+}
+
+#main {
+  position: relative;
+  text-align: center;
+  h1 {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.